drivers/hmis/lib/form_data/assessments/dummy_intake_assessment.json

```diff
@@ -0,0 +1,357 @@
+          "enable_when": [
+            {
+              "question": "4.11-2",
+              "operator": "EQUAL",
+              "answer_code": "1"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "type": "GROUP",
+      "link_id": "health-group",
+      "text": "Health",
+      "item": [
+        {
+          "type": "CHOICE",
+          "link_id": "4.04-2",
+          "text": "Covered by Health Insurance",
+          "pick_list_reference": "yesNoMissing"
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "health-insurance",
+          "text": "Health Insurance",
+          "pick_list_options": [
+            {
+              "code": "0",
+              "label": "Medicaid"
+            },
+            {
+              "code": "1",
+              "label": "Medicare"
+            }
+          ]
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "4.05-2",
+          "text": "Physical Disability",
+          "pick_list_reference": "yesNoMissing"
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "4.05-A",
+          "text": "Is this a disabling condition?",
+          "pick_list_reference": "yesNoMissing",
+          "enable_behavior": "ALL",
+          "enable_when": [
+            {
+              "question": "4.05-2",
+              "operator": "EQUAL",
+              "answer_code": "1"
+            }
+          ]
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "4.06-2",
+          "text": "Developmental Disability",
+          "pick_list_reference": "yesNoMissing"
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "4.07-2",
+          "text": "Chronic Health Condition",
+          "pick_list_reference": "yesNoMissing"
+        },
+        {
+          "type": "GROUP",
+          "link_id": "4.07-2-conditional",
+          "item": [
+            {
+              "type": "CHOICE",
+              "link_id": "4.07-A",
+              "text": "Is this a disabling condition?",
+              "pick_list_reference": "yesNoMissing",
+              "enable_behavior": "ALL",
+              "enable_when": [
+                {
+                  "question": "4.07-2",
+                  "operator": "EQUAL",
+                  "answer_code": "1"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "4.08-2",
+          "text": "HIV/AIDS",
+          "pick_list_reference": "yesNoMissing"
+        },
+        {
+          "type": "CHOICE",
+          "link_id": "4.09-2",
+          "text": "Mental Health Disorder",
+          "pick_list_reference": "yesNoMissing"
+        },
+        {
+          "type": "GROUP",
+          "link_id": "4.09-group",
+          "item": [
+            {
+              "type": "CHOICE",
+              "link_id": "4.09-A",
+              "text": "Is this a disabling condition?",
```

Comment:
Is this the new language for indefinite and impairing?

Reply:
This file in particular isn't really ready for review. I'm working on updating it today, will have another PR later!